vmware_tray/VMWAddOns.h

```diff
@@ -20,5 +20,6 @@
 #define CLIPBOARD_POLL 'clPo'
 #define CLOCK_POLL 'ckPo'
 #define SHRINK_DISKS 'shDk'
+#define TIME_SYNC_SHARING 'time'
```

Comment:
Should not be using only letters of the first word, collisions would be too easy